services/mesh.c

```diff
@@ -2050,6 +2068,13 @@ mesh_serve_expired_callback(void* arg)
 			}
 		}
 
+		/* Add EDE Stale Answer (RCF8914). Ignore global do_ede as this is
+		 * warning instead of an error */
+		if (r->edns.edns_present && qstate->env->cfg->serve_expired_ede) {
+			edns_opt_list_append_ede(&r->edns.opt_list_out,
+				mstate->s.region, LDNS_EDE_STALE_ANSWER, "");
```

Comment:
Why not NULL as in earlier cases in localzone.c?
```suggestion
			edns_opt_list_append_ede(&r->edns.opt_list_out,
				mstate->s.region, LDNS_EDE_STALE_ANSWER, NULL);
```

Reply:
both are fine, changing to NULL for consistency